fix(docs): Immich quota claim note (#8151)
immich-app/immich: docs/docs/administration/oauth.md

| Client ID                                            | string  | (required)           | Required. Client ID (from previous step)                                                                                  |
| Client Secret                                        | string  | (required)           | Required. Client Secret (previous step)                                                                                   |
| Scope                                                | string  | openid email profile | Full list of scopes to send with the request (space delimited)                                                            |
| Signing Algorithm                                    | string  | RS256                | The algorithm used to sign the id token (examples: RS256, HS256)                                                          |
| Storage Label Claim                                  | string  | preferred_username   | Claim mapping for the user's storage label                                                                                |

> | Storage Quota Claim                                  | string  | immich_quota         | Claim mapping for the user's storage<br />**Note:** Claim is only used on user creation and cannot be changed afterwards. |

Reviewer: ```suggestion
| Storage Quota Claim                                  | string  | immich_quota         | Claim mapping for the user's storage<br />**Note:** Claim is only used on user creation and not synchronized after that. |
```
